mangum/__init__.py

```diff
@@ -1 +1,5 @@
+from .response import Response
```

Comment:
Please move `Response` and `Scope` to a single file. The reason for doing so isn't for the imports, rather it is easier to maintain a single file with related classes like this.

Reply:
Since I was able to clean up adapter so much, they might fit nicely in there.